addons/etcd/templates/cmpv.yaml

```diff
@@ -6,12 +6,16 @@ metadata:
     {{- include "etcd.labels" . | nindent 4 }}
 spec:
   compatibilityRules:
-    - releases:
-        - v3.5.6
-      compDefs:
-        - etcd
+    - compDefs: [etcd-v3.5.6]
```

Comment:
Can etcd v3.5.6 and v3.5.15 share the same cmpd? If they can, keep only one cmpd to reduce maintenance costs.

Reply:
add a version check and merge them together